Sources/Splash/Grammar/SwiftGrammar.swift

```diff
@@ -47,10 +47,14 @@ public struct SwiftGrammar: Grammar {
             return false
         case (")", _):
             return false
-        case ("/", "/"), ("/", "*"):
+        case ("/", "/"), ("/", "*"), ("*", "/"):
             return true
         case ("/", _):
             return false
+        case ("(", _) where delimiterB != ".":
```

Comment:
This is needed to handle cases like `callAFunction(.withAnEnum)`.

Reply:
Makes sense!